wildfly-maven-plugin execute-commands fails on JDK 1.8.0_161

```
cd camel-cxf-jaxws-cdi-secure
mvn install -Pdeploy
...
Exception in thread "main" javax.xml.parsers.FactoryConfigurationError: Provider __redirected.__SAXParserFactory not found
        at javax.xml.parsers.FactoryFinder.newInstance(FactoryFinder.java:200)
        at javax.xml.parsers.FactoryFinder.newInstance(FactoryFinder.java:152)
        at javax.xml.parsers.FactoryFinder.find(FactoryFinder.java:232)
        at javax.xml.parsers.SAXParserFactory.newInstance(SAXParserFactory.java:127)
        at jdk.xml.internal.JdkXmlUtils.getSAXFactory(JdkXmlUtils.java:235)
        at jdk.xml.internal.JdkXmlUtils.<clinit>(JdkXmlUtils.java:78)
        at com.sun.org.apache.xerces.internal.impl.xs.opti.SchemaParsingConfig.<init>(SchemaParsingConfig.java:319)
        at com.sun.org.apache.xerces.internal.impl.xs.opti.SchemaParsingConfig.<init>(SchemaParsingConfig.java:254)
        at com.sun.org.apache.xerces.internal.impl.xs.traversers.XSDHandler.<init>(XSDHandler.java:480)
        at com.sun.org.apache.xerces.internal.impl.xs.traversers.XSDHandler.<init>(XSDHandler.java:487)
        at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaLoader.<init>(XMLSchemaLoader.java:357)
        at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaLoader.<init>(XMLSchemaLoader.java:292)
        at com.sun.org.apache.xerces.internal.jaxp.validation.XMLSchemaFactory.<init>(XMLSchemaFactory.java:97)
        at javax.xml.validation.SchemaFactoryFinder._newFactory(SchemaFactoryFinder.java:232)
        at javax.xml.validation.SchemaFactoryFinder.newFactory(SchemaFactoryFinder.java:146)
        at javax.xml.validation.SchemaFactory.newInstance(SchemaFactory.java:213)
        at __redirected.__SchemaFactory.<clinit>(__SchemaFactory.java:65)
        at __redirected.__JAXPRedirected.initAll(__JAXPRedirected.java:94)
        at org.wildfly.plugin.cli.ModuleEnvironment.initJaxp(ModuleEnvironment.java:69)
        at org.wildfly.plugin.cli.CommandExecutor.executeCommands(CommandExecutor.java:114)
        at org.wildfly.plugin.cli.CommandExecutor.execute(CommandExecutor.java:87)
        at org.wildfly.plugin.cli.CommandExecutor.execute(CommandExecutor.java:106)
        at org.wildfly.plugin.cli.ExecuteCommandsMojo.execute(ExecuteCommandsMojo.java:257)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: java.lang.ClassNotFoundException: __redirected/__SAXParserFactory
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:348)
        at javax.xml.parsers.FactoryFinder.getProviderClass(FactoryFinder.java:124)
        at javax.xml.parsers.FactoryFinder.newInstance(FactoryFinder.java:188)
        ... 44 more
```

Blocked by https://issues.jboss.org/projects/WFMP/issues/WFMP-94

--- a/pom.xml
+++ b/pom.xml
@@ -61,7 +61,7 @@
         <version-maven-source-plugin>2.3</version-maven-source-plugin>
         <version-maven-surefire-plugin>2.18.1</version-maven-surefire-plugin>
         <version-maven-war-plugin>3.0.0</version-maven-war-plugin>
-        <version-wildfly-maven-plugin>1.2.1.Final</version-wildfly-maven-plugin>
+        <version-wildfly-maven-plugin>1.2.2.Final</version-wildfly-maven-plugin>
         
         <deploy.skip>true</deploy.skip>
     </properties>

